.circleci/config.yml

```diff
@@ -108,9 +141,25 @@ commands:
             zip identity-idp-functions.zip identity-idp-functions.template.yaml
             zip identity-idp-functions.zip buildspec.yml
             zip identity-idp-functions.zip gitsha.txt
+  
+  copy-to-s3:
+    steps:
+      - run:
+          name: "Copy packages to S3"
+          command: |
+            cd source
             aws s3 cp identity-idp-functions.template.yaml s3://$DEPLOY_S3_BUCKET/circleci/identity-idp-functions/$CIRCLE_BRANCH/
             aws s3 cp identity-idp-functions.zip s3://$DEPLOY_S3_BUCKET/circleci/identity-idp-functions/$CIRCLE_BRANCH/
 
+  copy-to-s3-prod:
+    steps:
+      - run:
+          name: "Copy packages to S3 prod"
+          command: |
+            cd source
+            aws s3 cp identity-infra-functions.template.yaml s3://$prod_DEPLOY_S3_BUCKET/circleci/identity-infra-functions/$CIRCLE_BRANCH/ --profile prod
```

Comment:
is the variable `$PROD_DEPLOY_S3_BUCKET` or `$prod_DEPLOY_S3_BUCKET`? I'm not used to seeing mixed casing like this